core/src/main/java/com/hcl/ecomm/core/models/TitleModel.java

```diff
@@ -0,0 +1,61 @@
+package com.hcl.ecomm.core.models;
```

Comment:
@rakeshkumar-verma : why has the model been changed ?

Reply:
removed